packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -45,7 +46,7 @@ const GenericInput = ({
   const { formatMessage } = useIntl();
   const [showPassword, setShowPassword] = useState(false);
 
-  const CustomInput = customInputs ? customInputs[type] : null;
+  const CustomInput = customInputs ? customInputs[customFieldUid || type] : null;
```

Comment:
Do you mean on the Inputs component, doing:

```jsx
<GenericInput type={customFieldUid || type} />
```

If so I haven't checked but I'm afraid this could introduce behavior changes

Reply:
hmm ok I was thinking there, but returning early in the `inputType` useMemo could be an option too I guess

But surprised that wouldn't work though, sure looks likes doing the exact same thing but just a bit earlier...

Reply:
```js
  // line 88
  const inputType = useMemo(() => {
    if (customFieldUid) return customFieldUid
    return getInputType(type);
  }, [type, customFieldUid]);
```

Reply:
My concern was that changing the type would potentially affect the rest of the GenericInput logic, instead of just the CustomInput line. But actually, since we return early when it's a custom input, you're right we don't care about the rest of the logic